### PR TITLE
Add poll_candidate and poll_candidate_list.

### PR DESCRIPTION
**Pull request by reiterl**
Add models and actions and tests.
Update option.delete and poll delete test.

### CODE REVIEW

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Looks really good!

File: openslides_backend/action/actions/poll_candidate/create.py [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-)
+from ...generics.create import CreateAction
 from ...util.action_type import ActionType
 from ...util.default_schema import DefaultSchema
 from ...util.register import register_action
 
 
 @register_action("poll_candidate.create", action_type=ActionType.BACKEND_INTERNAL)
-class PollCandidateCreate(CreateActionWithInferredMeeting):
+class PollCandidateCreate(CreateAction):
```
Comment (jsangmeister) [inline openslides_backend/action/actions/poll_candidate/create.py:9]: Why did you change the base classes for the create actions? I think it was correct before

Comment (r-peschke) [reply openslides_backend/action/actions/poll_candidate/create.py:9]: Because it is not necessary to get the meeting_id from parent etc., it is set in payload.

Comment (jsangmeister) [reply openslides_backend/action/actions/poll_candidate/create.py:9]: But this way, it's redundant information. We get the meeting id from the parent in all other scenarios, so we should keep it this way.

**Review by jsangmeister — APPROVED** [review]
